client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss

```diff
@@ -3,6 +3,10 @@
 @import "jetpack-connect/colors.scss";
 
 .blog.blogger-intent {
+	.step-container__header {
+		margin: 0;
```

Comment:
It avoids this unnecessary scroll:

<img width="802" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/63ef2c71-efda-4ce0-bca3-16f4a000122b">